src/AppRouting.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route, Redirect } from 'react-router-dom';
+
+import { Trending, Favorite } from './components';
+
+const AppRouting = () => (
+  <Router>
+    <Switch>
+      <Route path="/trending" component={Trending} />
+      <Route path="/favorite" component={Favorite} />
+      <Redirect to="/trending" />
```

Comment:
It would be nice to send an object to let the user know what happened with that not found route.